### PR TITLE
[Merged by Bors] - chore: exactly 4 spaces in subsequent lines for def

### PR DESCRIPTION
**Pull request by mo271**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by mo271** [comment]
just as #7286 and #7296 but for defs

File: Mathlib/Algebra/Homology/HomologicalComplex.lean [hunk 1]
```diff
@@ -93,8 +93,8 @@ theorem ext {C₁ C₂ : HomologicalComplex V c} (h_X : C₁.X = C₂.X)
 #align homological_complex.ext HomologicalComplex.ext
 
 /-- The obvious isomorphism `K.X p ≅ K.X q` when `p = q`. -/
-def XIsoOfEq (K : HomologicalComplex V c) {p q : ι} (h : p = q) :
-  K.X p ≅ K.X q := eqToIso (by rw [h])
+def XIsoOfEq (K : HomologicalComplex V c) {p q : ι} (h : p = q) : K.X p ≅ K.X q :=
+ eqToIso (by rw [h])
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Homology/HomologicalComplex.lean:97]:
```suggestion
  eqToIso (by rw [h])
```

File: Mathlib/Algebra/Ring/Equiv.lean [hunk 2]
```diff
@@ -120,7 +120,7 @@ instance (priority := 100) toNonUnitalRingHomClass [NonUnitalNonAssocSemiring R]
 `RingEquiv`. This is declared as the default coercion from `F` to `α ≃+* β`. -/
 @[coe]
 def toRingEquiv [Mul α] [Add α] [Mul β] [Add β] [RingEquivClass F α β] (f : F) :
-  α ≃+* β :=
+    α ≃+* β :=
 { (f : α ≃* β), (f : α ≃+ β) with }
```
Comment (Ruben-VandeVelde) [inline Mathlib/Algebra/Ring/Equiv.lean:124]:
```suggestion
  { (f : α ≃* β), (f : α ≃+ β) with }
```

File: Mathlib/CategoryTheory/Adjunction/Limits.lean [hunk 3]
```diff
@@ -67,7 +67,7 @@ Auxiliary definition for `functorialityIsLeftAdjoint`.
 -/
 @[simps]
 def functorialityUnit :
-  𝟭 (Cocone K) ⟶ Cocones.functoriality _ F ⋙ functorialityRightAdjoint adj K where
+    𝟭 (Cocone K) ⟶ Cocones.functoriality _ F ⋙ functorialityRightAdjoint adj K where
     app c := { hom := adj.unit.app c.pt }
```
Comment (Ruben-VandeVelde) [inline Mathlib/CategoryTheory/Adjunction/Limits.lean:71]:
```suggestion
  app c := { hom := adj.unit.app c.pt }
```
(many times in this file, if you're up for it)

Comment (mo271) [reply Mathlib/CategoryTheory/Adjunction/Limits.lean]: will go through that file...

Comment (mo271) [reply Mathlib/CategoryTheory/Adjunction/Limits.lean]: done

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**